src/RSocketSetup.h

```diff
@@ -34,6 +34,7 @@ class RSocketSetup {
 
   void createRSocket(
       std::shared_ptr<RSocketResponder> requestResponder,
+      bool resumable = false,
```

Comment:
Should we have another approach to injecting this, or is this truly the only bool we'll ever pass in here? I don't want to accumulate a list of primitive args in this API.

For example, what about keepalive timeout config?